contrib/blueflood-docker/configs/ES-Setup/index_settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Does this need to be independent?  Is there a use case for having our docker example load up a different schema than our actually install?